campaigns/mobile/mobile-snippet-widget-200.html

```diff
@@ -327,6 +327,11 @@ <h2 class="thank-you hidden" id="thank-you-message-{{ snippet_id }}">{{ thank_yo
          var button = document.querySelector('#firefox-mobile-activation-{{ snippet_id }} .submit-button');
          var input_type;
 
+         // Bug 1238092. Can be removed when Bug 1238092 gets fixed.
+         input.addEventListener('keypress', function(event) {
+             event.stopPropagation();
```

Comment:
ah, so we just want these keypress event to not bubble and thus not hit the core Fx code that would trap it and send it to the search bar. Correct? Then after whichever form of a fix lands in release, we can remove this? Although, it will then likely still be broken in older versions so, we will probably want to have this hanging around for a while.